### ISSUE TITLE
flask run not work

### ISSUE THREAD

**Issue by protream** [pallets/flask#2158]
I have set `FLASK_APP=manage.py', but got：

```
Error: The file/path provided (manage) does not appear to exist.  Please verify the path is correct.  If app is not on PYTHONPATH, ensure the extension is .py
```

**Comment by kblomqvist** [comment]
Most likely you have ImportError somewhere, which you can't see because it is not propagated. Install flask from master and you can see what's the problem. This was fixed in #1950.

**Comment by protream** [comment]
@kblomqvist There is a ImportError indeed. Thank you!
